MiniApp/Classes/core/Storage/MAManifestStorage.swift

```diff
@@ -2,58 +2,133 @@ import Foundation
 import CommonCrypto
 
 internal class MAManifestStorage {
```

Comment:
May be renaming them to `MAManifestJSONStorage` or some other self descriptive name ?